package.json

```diff
@@ -2338,7 +2338,7 @@
 							"string"
 						],
 						"default": null,
-						"markdownDescription": "The path to a log file for very detailed logging in the Dart Analysis Server that may be useful when trying to diagnose Analysis Server issues. Use `${workspaceName}` to insert the name of the current workspace in the file path.",
+						"markdownDescription": "The path to a log file for very detailed logging in the Dart Analysis Server that may be useful when trying to diagnose Analysis Server issues. Use `${name}` in the log file name to insert the Debug Session name to prevent concurrent debug sessions overwriting each others logs. Use `${workspaceName}` to insert the name of the current workspace in the file path.",
```

Comment:
`${name}` isn't supported here, it's only supported in the loges that are debug-session specific (that is DAP, DartTest, FlutterRun, FlutterTest, WebDaemon).

I think as of the current master version, the descriptions should all be accurate.